package.json

```diff
@@ -14,27 +14,21 @@
     "coverage": "npm run pretest && nyc floss --path=test && nyc report --reporter=html",
     "coverage:ci": "npm run pretest && nyc floss --path=test && nyc report --reporter=text-lcov > coverage.lcov",
     "docs": "mkdirp dist && webdoc -R README.md",
-    "pretypes": "npm run compile:docs",
-    "types": "npm run types:legacy && npm run types:default",
-    "types:default": "mkdirp dist/types && jsdoc -c types/jsdoc.conf.json && node types/assemble pixi.js",
-    "types:legacy": "mkdirp dist/types && jsdoc -c types/jsdoc-legacy.conf.json && node types/assemble pixi.js-legacy",
-    "posttypes": "rimraf dist/compiled",
     "lint": "eslint --ext .js --ext .ts test bundles packages tools --ignore-path .gitignore --max-warnings 0",
     "lintfix": "npm run lint -- --fix",
     "prebuild": "npm run clean:build",
     "build": "npm run build:dev",
     "build:prod": "cross-env NODE_ENV=production rollup -c",
     "build:dev": "rollup -c",
-    "postbuild": "npm run types",
+    "build:types": "rm -r out; tsc; api-extractor-lerna-monorepo; rm -r out",
```

Comment:
This doesn't seem to work for me on windows as `rm` is not a command

Can this be replaced with using `rimraf`
 `"build:types": "rimraf out && tsc && api-extractor-lerna-monorepo && rimraf out",`